test/e2e/framework/deployments.go

```diff
@@ -47,7 +47,7 @@ func (f *Framework) NewNetShootDeployment(cluster ClusterIndex) *corev1.PodList
 					Containers: []corev1.Container{
 						{
 							Name:            "netshoot",
```

Comment:
Can we change the container name too, or would that break the tests?

Reply:
I don't know since theres no local tests that use it.
Anyhow I don't think it's critical we didn't change the name of the yaml resource.. Maybe a nice to have..

Reply:
I want to get rid of this, we can just use the standard connectivity framework, or.. in the case where this is used (lighthouse) just dns queries, we don't actually need to check connectivity there

Reply:
OK so until you get rid of it, shouldnt we at least use the smaller image?

Reply:
Yes yes, I'm fine with it, let's leave the name alone just in case, I didn't look at the implementation (that's what I mean, that I don't have time to look at it specially if I plan to remove it all together)